drivers/hud_path_report/app/models/hud_path_report/filters/path_filter.rb

```diff
@@ -10,13 +10,12 @@ def default_project_type_codes
       [:so, :services_only]
     end
 
-    def project_options_for_select(user:)
-      path_funded = ::GrdaWarehouse::Hud::Funder.funding_source(funder_code: 21)
-      all_project_scope.joins(:funders).options_for_select(user: user, scope: path_funded)
-    end
-
     def path_project_types_for_select
       GrdaWarehouse::Hud::Project::PROJECT_GROUP_TITLES.select { |k, _| k.in?([:so, :services_only]) }.invert.freeze
     end
+
+    def path_project_type_ids
+      [:so, :services_only].map { |s| GrdaWarehouse::Hud::Project::PERFORMANCE_REPORTING[s] }.flatten
```

Comment:
It looks like we do this a bunch of times too `[:so, :services_only]`.  Maybe we should move that to a constant on Project too?

Reply:
Ok I will move them both to `Project`